### ISSUE TITLE
Selection of autocomplete doesn't work on mobile (iOS Safari)

### ISSUE THREAD

**Issue by AlexandreKilian** [kenny-hibino/react-places-autocomplete#92]
due to this part here, only mouse events are valid for selection

```
onMouseOver={() => this.setActiveItemAtIndex(p.index)}
onMouseDown={() => this.selectAddress(p.suggestion, p.placeId)}
```

changing `onMouseDown` to `onClick` or adding `onTouchEnd` and `onTouchStart` events should fix this.

**Comment by AlexandreKilian** [comment]
I added a pull request for this:

https://github.com/kenny-hibino/react-places-autocomplete/pull/93

**Comment by kenny-hibino** [comment]
Closed via #93
